packages/boot/src/__tests__/acceptance/application-metadata.booter.acceptance.ts

```diff
@@ -36,6 +36,7 @@ describe('application metadata booter acceptance tests', () => {
     app = new MyApp({
       rest: givenHttpServerConfig(),
     });
+    await app.start();
```

Comment:
I am confused. Why do we need to start the app? I don't see any change in the tests above. Is this need for a change a signal that you are introducing a breaking change, i.e. existing LB4 projects can break after upgrading to a newer version of LB4 runtime?

Reply:
We can revert this.